barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.test.cpp

```diff
@@ -541,61 +541,46 @@ TEST_F(ClientIVCTests, StructuredTraceOverflow)
 };
 
 /**
- * @brief Test dynamic structured trace overflow block mechanism
- * @details Tests the case where the required overflow capacity is not known until runtime. Accumulates two circuits,
- * the second of which overflows the trace but not enough to change the dyadic circuit size and thus there is no need
- * for a virtual size increase of the first key.
+ * @brief Test the structured trace overflow mechanism in a variety of different scenarios
  *
  */
-TEST_F(ClientIVCTests, DynamicOverflow)
+TEST_F(ClientIVCTests, DynamicTraceOverflow)
```

Comment:
Just reworking this test and the next into a single test with some additional cases, including one that would have caught the bug